client/my-sites/post-share/index.jsx

```diff
@@ -0,0 +1,273 @@
+	};
+
+	state = {
+		skipped: PostMetadata.publicizeSkipped( this.props.post ) || [],
+		message: PostMetadata.publicizeMessage( this.props.post ) || this.props.post.title
+	};
+
+	hasConnections() {
+		return !! ( this.props.connections && this.props.connections.length );
+	}
+
+	toggleConnection = id => {
+		const skipped = this.state.skipped.slice();
+		const index = skipped.indexOf( id );
+		if ( index !== -1 ) {
+			skipped.splice( index, 1 );
+		} else {
+			skipped.push( id );
+		}
+
+		this.setState( { skipped } );
+	};
+
+	skipConnection( { keyring_connection_ID } ) {
+		return this.state.skipped.indexOf( keyring_connection_ID ) === -1;
+	}
+
+	isConnectionActive = connection =>
+		connection.status !== 'broken' && this.skipConnection( connection );
+
+	renderServices() {
+		if ( ! this.props.site || ! this.hasConnections() ) {
+			return;
+		}
+
+		return this.props.connections.map( connection =>
+			<Connection
+				connection={ connection }
+				onToggle= { this.toggleConnection }
+				isActive={ this.isConnectionActive( connection ) }
+				key={ connection.keyring_connection_ID }
+			/>
+		);
+	}
+
+	setMessage = message => this.setState( { message } );
+
+	renderMessage() {
+		if ( ! this.hasConnections() ) {
+			return;
+		}
+
+		const targeted = this.hasConnections() ? this.props.connections.filter( this.isConnectionActive ) : [];
+		const requireCount = includes( map( targeted, 'service' ), 'twitter' );
+		const acceptableLength = ( requireCount ) ? 140 - 23 - 23 : null;
+
+		return (
+			<PublicizeMessage
+				message={ this.state.message }
+				preview={ this.props.post.title }
+				requireCount={ requireCount }
+				onChange={ this.setMessage }
+				acceptableLength={ acceptableLength } />
+		);
+	}
+
+	dismiss() {
+		this.props.dismissShareConfirmation( this.props.siteId, this.props.post.ID );
+	}
+
+	sharePost = () => {
+		this.props.sharePost( this.props.siteId, this.props.post.ID, this.state.skipped, this.state.message );
+	};
+
+	isButtonDisabled() {
+		if ( this.props.requesting ) {
+			return true;
+		}
+
+		return this.props.connections.filter( this.isConnectionActive ).length < 1;
+	}
+
+	render() {
+		if ( ! this.props.isPublicizeEnabled ) {
+			return null;
+		}
+
+		if ( ! this.props.planHasRepublicizeFeature ) {
+			return ( <div className="post-share">
+				<Banner
+					feature="republicize"
+					title={ this.props.translate( 'Unlock the ability to re-share posts to social media' ) }
+					callToAction={ this.props.translate( 'Upgrade to Premium' ) }
+					description={ this.props.translate( 'Get unlimited premium themes, video uploads, monetize your site and more.' ) }
+				/>
+			</div> );
+		}
+
+		const classes = classNames( 'post-share__wrapper', {
+			'has-connections': this.hasConnections()
+		} );
+
+		return (
+			<div className="post-share">
+				{ this.props.requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ this.props.translate( 'Sharing…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sharing...' )` *ES Score: 17*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/post-share-janitorial)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).